spec/support/vcr_cassettes/mobile/profile/v2/get_phone_status_incomplete.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
   - request:
       method: get
-      uri: <VETS360_URL>/contact-information-hub/cuf/contact-information/v2/1/telephones/status/c3c6502d-f660-409c-9bc9-a7b7ce4f0bc5
+      uri: <VETS360_URL>/contact-information-hub/contact-information/v2/2.16.840.1.113883.4.349/123498767V234859%5ENI%5E200M%5EUSVHA
       body:
```

Comment:
Wrong URI. A transaction_id is required in the uri. 
Should be something like this:
      uri: <VETS360_URL>/contact-information-hub/contact-information/v2/addresses/status/#{transaction_id}

The following uri only gathers the user info, which is not what the VCR response is supposed to do.
<VETS360_URL>/contact-information-hub/contact-information/v2/2.16.840.1.113883.4.349/123498767V234859%5ENI%5E200M%5EUSVHA